server/db/db.js

```diff
@@ -169,6 +185,7 @@ const teardown = () =>
   .then(() => bookshelf.knex.schema.dropTable('contact_lists'))
   .then(() => bookshelf.knex.schema.dropTable('contacts'))
   .then(() => bookshelf.knex.schema.dropTable('users'))
+  .then(() => bookshelf.knex.schema.dropTable('campaigns_users'))
```

Comment:
We will likely need to drop this table first since it depends on both campaigns and users. I may be wrong. Have you tested this?

Reply:
The test for this will be in my ticket since I'm finishing out the entire life cycle of the feature.